### PR TITLE
Update SAML-autnetication-org-management.md

### PR DESCRIPTION
**Pull request by lemonez**
Whoever reviews this, please take a look at the wording where I've changed things. This is outside of my area of expertise so i'm not 100% on the terminology.

Just fixing some wording and formatting. No associated Jira ticket.

## Checklist
- [ ] My PR title includes a relevant Jira ticket name
- [x] If no associated Jira ticket, my PR title is informative (individual commit messages are squashed in this repo)
- [x] Unnecessary text (Notes, TODOs, etc.) has been removed
- [ ] Screenshots, if present, are full-width and follow the [Screenshot guidelines](https://algorithmia.atlassian.net/wiki/spaces/CUSTOMERS/pages/1634861478/CFD+Style+Guide#Screenshots)
- [x] Grammar, style, tone, punctuation, etc. follow the Algorithmia [manual of style](https://docs.google.com/document/d/1PPVfgMkX7-EVGLPMhN1E485CAXu9QfhSLKM0lZhnZdU/edit?usp=sharing)
- [x] Headers are “Sentence case with Proper Nouns capitalized”
- [ ] Placeholder strings are UPPER_SNAKE_CASE and are named as in the [Glossary](https://docs.google.com/document/d/1bYs0j_a4v8LbkbS8r0x2SZ_J6mZ4sIt4w_60dq8iGh4/edit#heading=h.qov2yks2a685) where appropriate

### CODE REVIEW

**Review by warmbowski — DISMISSED** [review]
This seems pretty straight forward edits. I'm not 100% on SAML terminology either, but your edits just seem to clairify, and not alter content.

**Comment by lemonez** [comment]
@warmbowski yes I actually worked with Blake on this, so I'm pretty confident, but I'll have him give another look too. thanks.

**Comment by lemonez** [comment]
(worked with blake after i made that comment)

**Comment by lemonez** [comment]
@blakerithmia could you please do a final review of changes here? we talked about some of them but want to make sure

File: _pages/platform/SAML-authentication-org-management.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+SAML authentication allows a user to use SAML assertions in order to log in to the platform. If configured, permission tags can be extracted from the SAML assertion and can be used to facilitate external management of organization membership and user security roles.
+
+This feature is available to [Algorithmia Enterprise](/enterprise) users who have configured SAML and external organization management functionality only.
+{: .notice-enterprise}
+
+#### Use cases
+-  Allowing user login through SAML
+-  Using an external management platform for user permission (admin/sudo)
+-  Using an external management platform for granting/revoking organization membership/administration privileges
+-  Creating local users that represent validated users verified through SAML
```
Comment (blakerithmia) [inline _pages/platform/SAML-authentication-org-management.md:26]: technical term would be authenticated through SAML, just figured id mention that incase it mattered, not sure if a change is needed

**Review by blakerithmia — DISMISSED** [review]

File: _pages/platform/SAML-authentication-org-management.md (hunk 1 above)
Comment (lemonez) [reply _pages/platform/SAML-authentication-org-management.md:26]: will make this change, thanks

**Pull request merged**